### PR TITLE
make code fancy and suggestions distinguishable

### PR DESCRIPTION
**Pull request by zardoy**
This is just a quick fixup. This PR includes only visual changes, hope to see them soon, as they're extremely important for me.

The completion constructor needed latest version of vscode typings so I added `@types/vscode` and removed the postinstall hook that was responsible for fetching typings.

Screenshot 2021-12-15 213740

The only downside of using syntax highlighting is that we losing word wrap, but I don't think it rly matters since it works only with builtin snippets

### CODE REVIEW

**Comment by ipatalas** [comment]
Hi! That looks really great. I haven't been around for a while but I'm back and keen to release a new version including your changes :)
Can you please rebase your PR with most recent changes? I did some updates in result of merging one of previous PRs and there are quite some conflicts.

File: test/extension.singleline.test.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { runTest as Test, runTestQuickPick as QuickPick } from './runner'
 
 const config = vsc.workspace.getConfiguration('postfix')
 
-describe('Single line template tests', () => {
+describe.only('Single line template tests', () => {
```
Comment (ipatalas) [inline test/extension.singleline.test.ts]: Don't forget to remove this `.only` ;)

Comment (ipatalas) [reply test/extension.singleline.test.ts]: Oh, it was actually my fault... when you refresh your branch from latest `develop` this should disappear.

**Comment by zardoy** [comment]
Hey! I just rebased it on the lateset changes from develop, but it seems already was merged. If it really was I think we can close this PR.

**Comment by ipatalas** [comment]
Yeah, I managed to merge it manually by cherry-picking your commit as there's been some noise as well which I did not want to merge.

**Pull request closed without merging**